c/serializer/samples/remote_monitoring/remote_monitoring.c

```diff
@@ -20,16 +24,32 @@ static const char* hubSuffix = "[IoTHub Suffix, i.e. azure-devices.net]";
 // Define the Model
 BEGIN_NAMESPACE(Contoso);
 
+#ifndef WINCE
 DECLARE_STRUCT(SystemProperties,
     ascii_char_ptr, DeviceID,
     _Bool, Enabled
+	Bool, Enabled
 );
+#else
+DECLARE_STRUCT(SystemProperties,
+ascii_char_ptr, DeviceID,
+bool, Enabled
```

Comment:
I've been struggling with bool because VS2013 compiler (at least for Windows Embedded Compact) does not recognize it as a native type. I did not check crt_abstractions.h, will look into it to see if it can be used for a more elegant workaround.